python_modules/dagster-test/dagster_test/graph_job_op_toys/many_events.py

```diff
@@ -71,8 +71,8 @@ def input_name_for_raw_file(raw_file):
 
 
 @op(
-    ins={"start": In(Nothing)},
-    out=Out(dagster_type=Nothing),
+    ins={"start": In(Nothing)},  # type: ignore
+    out=Out(dagster_type=Nothing),  # type: ignore
```

Comment:
This is down to an obscure `mypy` bug. `mypy` for some reason thinks `Nothing` is of type `builtins.object`, which cause these lines to type-error. I'm 95% sure this is because `mypy` has some buggy special-case logic somewhere for the name `builtins`, and `Nothing` originates in `dagster.builtins`.

Pyright doesn't have this problem and correctly types the code and finds no error here. One more reason to switch to pyright.

Reply:
Frustrating. Did you try renaming `dagster.builtins` to something else? This may effect a reasonable number of users so is worth resolving if possible.

Reply:
I fixed this by annotating `Nothing` at the source in a downstack PR. Turned out not to be an issue with the name `builtin`, but rather another quirk (bug?) of `mypy` with inconsistent type inference of `type(None)`.